crates/iota-types/src/iota_system_state/mod.rs

```diff
@@ -357,6 +357,17 @@ where
                     })?;
             Ok(validator.into_iota_validator_summary())
         }
+        2 => {
+            let validator: ValidatorV2 =
+                get_dynamic_field_from_store(object_store, versioned.id.id.bytes, &version)
+                    .map_err(|err| {
+                        IotaError::IotaSystemStateRead(format!(
+                            "Failed to load inner validator from the wrapper: {:?}",
+                            err
```

Comment:
Always a bit nicer to have variables inside if possible
```suggestion
                            "Failed to load inner validator from the wrapper: {err:?}"
```